### DISCUSSION TITLE
Input not editable with 1.8.*

### DISCUSSION THREAD

**Discussion by danielefederico** [mathuo/dockview#404] [General]
Hello everyone,
sorry for the silly question, but I was wondering if anything has changed in 1.8.* that would prevent my inputs to become editable.

My inputs used to work fine with 1.7.5, but since I upgraded to 1.8.5, they don't work anymore. I'm not able to edit their values.
Everything go back to normal if I revert to 1.7.5. 

Given the fact my code works even in a separate application, I was wondering if I am missing something trivial.

Thanks!

**Comment by mathuo** [comment]
Are you able to provide any more information on the issue, or perhaps even a small example by adjusting the following sandbox (you can also adjust the dockview version being used here)

https://codesandbox.io/s/github/mathuo/dockview/tree/master/packages/docs/sandboxes/simple-dockview
